notebooks/en/analyzing_art_with_hf_and_fiftyone.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>a "." at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>